ui/src/routes/admin/users/create-user/index.tsx

```diff
@@ -56,42 +65,85 @@ const formSchema = z.object({
   email: asOptionalField(z.string().email()),
   password: asOptionalField(z.string().min(1)),
   temporary: z.boolean(),
+  organizations: z.array(z.string()).min(1, {
+    message:
+      'You must grant the user READERS access to at least one organization',
```

Comment:
I would not emphasize / mention the "READERS" part at all here, to not give the user the impression that a user could also be created as part of the WRITERS group, for example; the READERS group is implicit on creation.

So again, I propose to be short and just say "The user must be part of at least one organization."